src/Server.ts

```diff
@@ -8,19 +10,29 @@ class Server {
         this.app = express();
     }
     bootstrap() {
+        this.initBodyParser();
         this.setupRoutes();
+
         return this;
     }
     run = () => {
         const { app, config: { port, env } } = this;
-        app.listen(port, (err) => {
+        app.listen(9000, (err) => {
```

Comment:
don't hard code the port call that from .env file only

Reply:
Done